testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/suite/suites/SuiteOauth2.java

```diff
@@ -35,6 +36,9 @@
                         .withGroups("oauth2")
                         .build(),
                 testOnEnvironment(EnvSinglenodeOauth2HttpProxy.class)
```

Comment:
so maybe now we can remove non secured proxy?

Reply:
I guess it better to run the tests on both the Environment (with non-secure and secure proxy).